roles/splunk_common/tasks/provision_apps_local.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Can we reused the existing https://github.com/splunk/splunk-ansible/blob/develop/roles/splunk_common/tasks/provision_apps.yml, but modify it so that it accepts a variable as an input? Then we can pass in either `splunk.apps_location_local` or `splunk.apps_location` and reduce playbook duplication.

Reply:
The local version of this basically the same.  I'll adjust the calls to the provision_apps.yml to handle either list.